### ISSUE TITLE
TAD wird auch für Monographien ausgelöst

### ISSUE THREAD

**Issue by IxTheoKm** [ubtue/ixTheo#431]
Der _Aufsatz_dienst wird auch für _Monographien_ ohne Einzelbeiträge ausgelöst!
Bsp.:
https://ixtheo-test.de/Record/477255485
Formular des TAD: " Aus der Zeitschrift „Islam and cultural change in Papua New Guinea“ mit der Signatur „57 A 601“ ... "
Es handelt sich aber um eine Monographie ohne Einzelbeiträge!

Wenn überhaupt Monographien berücksichtigt werden sollen, dann nur diejenigen, die auch Aufsätze enthalten. Das ist allerdings nicht in jedem Fall eindeutig. 
- Festschriften und Kongressschriften sind i.d.R. Aufsatzsammlungen. 
- Zusätzlich diejenigen Monographien, die als Schlagwort oder "Art des Inhalts" den Begriff Aufsatzsammlung enthalten. 
- Zusätzlich diejenigen Monographien, mit denen Aufsätze verknüpft sind.

Bei allen anderen Monographien kann man m.E. den TAD nicht anbieten.

**Comment by mtrojan-ub** [comment]
Wie telefonisch besprochen, TAD anzeigen wenn:
- Medienart: "Festschrift", oder
- Schlagworte: "Konferenzschrift", "Kongress" oder "Aufsatzsammlung"

oder auch wenn
- mindestens ein untergeordnetes Werk mit Medienart "Aufsatz" existiert

**Comment by mtrojan-ub** [comment]
Hinweis: Für Schlagworte wird die allgemeine Funktion aus SolrMarc verwendet, die folgende Felder umfasst:
'600', '610', '611', '630', '648', '650', '651', '653', '655', '656'

**Comment by mtrojan-ub** [comment]
Nach Rücksprache IxTheoKM / jriedl wird nochmal geklärt, ob es nicht evtl sinnvoller wäre den TAD direkt nur für Aufsätze aufrufbar zu machen

**Comment by mtrojan-ub** [comment]
Besprochen im Meeting heute:
TAD wird aktuell erstmal nur auf auf die Medienart "Aufsatz" beschränkt.
Nutzung des TAD speziell für Festschriften/Kongresschriften/Aufsatzsammlungen muss erst noch diskutiert werden.

**Comment by socheres** [comment]
Im alten IxTheo konnte man alle Artikel/Beiträge aus Zss./Sammelbänden über TAD bestellen.
Z.B. erscheint der TAD-Button im folgenden Bsp. nicht: https://ixtheo.de/Record/464987903

**Comment by IxTheoKm** [comment]
#thefass will noch weitere Anmerkungen ergänzen. 

Hier in seinem Auftrag eine Liste der möglichen Kriterien für Sammelbände für den TAD (Bestand DE-21):
_Noch zu klären: soll das auch für Online-Publikationen gelten?_ 
Bsp.: 487709306 + 483609951

[Sammelbände.pdf](https://github.com/ubtue/ixTheo/files/1055365/Sammelbande.pdf)

**Comment by IxTheoKm** [comment]
Ergänzung nach Rücksprache mit dem BSZ:
Alle gedruckten Aufsätze (Satztyp Ao... in PICA) sind erkennbar durch folgende Kriterien:

Marc-Feld 007: "**t**" an Position 1 = gedrucktes Werk
(Online-Publikationen sollen nach Rücksprache mit thefass nicht berücksichtigt werden.)

Im Header: "**a**" an Position 7 = unselbständiges Werk.
(Derzeit gibt es nur "a"; 
später soll zwischen "a" und "b" unterschieden werden: 
unselbständiges Werk in einer Zeitschrift oder unselbständiges Werk in einer Monographie.)

**Comment by thefass** [comment]
@mtrojan-ub : Bitte als ersten Schritt den Kommentar von @IxTheoKm umsetzen.
Auf diese Weise werden alle Aufsätze von Zeitschriften, aber auch von Sammelwerken, die einzeln katalogisiert sind, für den TAD nutzbar.

**Comment by thefass** [comment]
Die obige Regel muss noch ergänzt werden
TAD kann nur ausgeführt werden, wenn die UB Bestand hat.
Prüfung über das Sigel der UB = 21 (exakt 21, denn es gibt auch Sigel wie 21/31)
PICA 7100 $B21
Marc 852 $a21
@IxTheoKm : bitte prüfen, ob das so stimmt

**Comment by IxTheoKm** [comment]
Das Sigel steht in der Regel nur in den _übergeordneten_ Aufnahmen der Zeitschriften oder Monographien. Die Aufsätze haben i.d.R. keine Lokaldaten der UB-Tübingen.

In PICA ist das Sigel der UB Tübingen in 7100 $B**21**, das ist korrekt.
In Marc ist es **852 $aDE-21** 
(für die Institute ist es in PICA z.B. 7100 $B21/31 und in Marc 852 $aDE-21-31)

Bsp.:
**278242650: Zeitschriftengesamtaufnahme**
...
$0000 xxxxxcy a22 3n 4500
$0001 565790668
$0003 DE-576
$0004 278242650
$0005 20080313153156
$0008 080304||||||||||||||||ger|||||||
$0852 $81
**$0852 $aDE-21**
$0852 1$cZA 10509$900
$085901$81.1$a2$i2008
$086630$a2.2008 -
$0866 0$zZLS-A: theol / ZLS-WZ
$0000 xxxxxcy a22 3n 4500
$0001 565790684
$0003 DE-576
$0004 278242650
$0005 20080414083532
$0008 080304||||||||||||||||ger|||||||
$0852 $81
$0852 $aDE-21-35
$0852 1$cZk 517$900
$085901$81.1$a2$i2008
$086630$a2.2008 -
$0000 xxxxxnx a22 zn 4500
$0001 567752437
$0003 DE-576
$0004 278242650
$0005 20080414083532
$0008 080414||||||||||||||||ger|||||||
$0852 $zBestand: 2.2008 -
$0852 $aDE-21-35
$0852 1$cZk 517$mp$900
$086630$xKV : Reihe abonniert
$0935 $aALKV$akneu$aabok
...

**356822567: Monographie (Festschrift)**
...
$0000 xxxxxcx a22 zn 4500
$0001 703755331
$0003 DE-576
$0004 356822567
$0005 20120214143927
$0008 120210||||||||||||||||ger|||||||
**$0852 $aDE-21**
$0852 1$c52 A 1652$900
$0935 $atheo
$0936ln$at4.9
$0938 $a1202$f2
$0000 xxxxxcx a22 zn 4500
$0001 861786955
$0003 DE-576
$0004 356822567
$0005 20150831114354
$0008 150826||||||||||||||||ger|||||||
$0541 $e13/806
$0852 $aDE-21-35
$0852 1$cOh 12.9$mp$900
$0938 $a1509$fK10
...

      
[In der SWB-Online-Hilfe steht, dass die Auslieferung von PICA 7100 $B im Marc-Feld 924 $b und 924 $c erfolgt. Das kann ich nicht nachvollziehen. Evtl. veraltet? Werde ich klären.]

**Comment by ruschein** [comment]
@IxTheoKm Ihrem letzten Kommentar zufolge wollten Sie etwas klären.  Hatten Sie da mittlerweile eine Gelegenheit das zu tun?

**Comment by IxTheoKm** [comment]
Ja, der MARC-Export des Sigels erfolgt wie oben beschrieben in 852 $a. Das fehlte in der SWB-Online-Hilfe und wurde nun nachgetragen.

In 924 werden weitere Informationen ausgeliefert. Da habe ich dann aber keine weiteren, detaillierten Angaben erfragt. Relevant ist ja das Sigel.
